### PR TITLE
Added learn more link to hoc courses

### PR DESCRIPTION
**Pull request by caleybrock**
New preferred style: 
screen shot 2015-09-15 at 1 11 40 pm

alt style:
screen shot 2015-09-15 at 10 54 34 am

Added the 'Learn more' link.

@tanyaparker please review.

### CODE REVIEW

**Comment by tanyaparker** [comment]
image

Can you make the space between 1) the text and the courses and 2) the button and the courses the same?

File: pegasus/sites.v3/code.org/public/teacher-dashboard/landing.haml [hunk 1]
```diff
@@ -76,7 +76,10 @@ theme: none
       %h1{style: 'margin-bottom: 0px'}= I18n.t('dashboard_landing_available_title')
       %h2= I18n.t('dashboard_hoc_courses')
       = I18n.t('dashboard_hoc_courses_desc')
-    %br/  
+      %div{style: 'float:right; margin-right:30px; margin-top:-8px;'}
+        %a{:href=>'https://hourofcode.com'}
+          %button= I18n.t('learn_more')
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/teacher-dashboard/landing.haml]: I think the text should be more descriptive like "Learn more about the Hour of Code" or "Join the Hour of Code"

Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/teacher-dashboard/landing.haml]: Show Roxanne your screenshot and she can suggest the better button text.

**Comment by caleybrock** [comment]
screen shot 2015-09-15 at 10 52 57 am

Updated the margins.

**Pull request merged**